# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: e95ff11012c5
# Full Hash: e95ff11012c57dd67f15d0d8a986969bc1c1a61e
# Author: Max Inden <mail@max-inden.de>
# Date: 2024-07-30 03:02:49
# Regressor Bug: 1910360
# File Overlap Count: 1
# Description:
#   Bug 1910360 - use quinn-udp instead of NSPR for QUIC I/O on Nightly r=kershaw
#   
#   https://phabricator.services.mozilla.com/D212959 adds optional support for QUIC
#   I/O via quinn-udp instead of NSPR. This is controlled via pref
#   `network.http.http3.use_nspr_for_io`. The default pref value was `true`, thus
# ==============================================================================

diff -r 59644f1ab054 -r e95ff11012c5 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Mon Jul 29 17:33:11 2024 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Mon Jul 29 17:34:41 2024 +0000
@@ -13556,7 +13556,7 @@
 # Use NSPR for HTTP3 UDP IO
 - name: network.http.http3.use_nspr_for_io
   type: RelaxedAtomicBool
-  value: true
+  value: @IS_NOT_NIGHTLY_BUILD@
   mirror: always
 
 - name: network.http.http3.enable_qlog
